<?xml version="1.0" encoding="UTF-8"?>

<launch>
    <arg name="initial_x" default="0" />
    <arg name="initial_y" default="0" />
    <arg name="initial_z" default="0" />
    <arg name="des_x" default="0" />
    <arg name="des_y" default="-4" />
    <arg name="des_z" default="-4" />
    <param name="initial_x" value="$(arg initial_x)" />
    <param name="initial_y" value="$(arg initial_y)" />
    <param name="initial_z" value="$(arg initial_z)" />
    <param name="des_pos_x" value="$(arg des_x)" />
    <param name="des_pos_y" value="$(arg des_y)" />
    <param name="des_pos_z" value="$(arg des_z)" />
    <node pkg="vitarana_drone" type="follow_wall.py" name="wall_follower" />
    <node pkg="vitarana_drone" type="go_to_point.py" name="go_to_point" />
    <node pkg="vitarana_drone" type="bug2.py" name="bug2" output="screen" />
</launch>

